benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -461,19 +461,23 @@ msgstr "Try again"
 
 #: benefits/enrollment/views.py:147 benefits/enrollment/views.py:149
 msgid "enrollment.pages.success.title"
-msgstr "Success!"
+msgstr "Success! Your discount is now linked to your bank card."
 
 #: benefits/enrollment/views.py:150
 msgid "enrollment.pages.success.p1"
-msgstr ""
-"Your discount is now linked to your bank card. You were not charged anything "
-"today."
+msgstr "You were not charged anything today."
 
 #: benefits/enrollment/views.py:150
 msgid "enrollment.pages.success.p2"
-msgstr ""
-"When boarding transit, pay with this same card, and your discount will "
-"automatically be applied to your fare purchase."
+msgstr "When boarding transit, pay with this same card, and your discount will automatically be applied to your fare purchase."
+
+msgid "enrollment.pages.success.p3"
+msgstr "For additional information on what to expect when using your connected bank card, please reach out to your transit provider, or check out our "
+"<a "
+"href=\"/help\" rel=\"noopener noreferrer\">help page</a>."
```

Comment:
Agreed, thanks for pointing this out! We always want to opt to generate URLs for our app using the `reverse("app:route")` pattern rather than hardcoding, if possible.

Reply:
What's the way to ensure the Help page's Back button properly brings users back to the page they were on? On this PR, clicking Help is bringing them to the Eligibility Start page.

Reply:
What page should they be on? 

The back button is generated using the session `origin` as the location, so each page the user advances needs to `session.update(request, origin="...")` assuming the user should go back to that page.

Reply:
@machikoyasuda It sounds like you're saying that if the user were to click the `help page` link on the Enrollment Success page and then click the "Go back" button on the Help page, they'd expect to go back to the Enrollment Success page?

@machikoyasuda and @thekaveman Looks like `enrollment/views.py` currently does not set the session `origin` in any of its view functions, so that explains why the last origin set is for the Eligibility Start page. Maybe we can make a separate ticket this?